jujugui/static/gui/src/app/components/boolean-config/boolean-config.js

```diff
@@ -93,22 +90,21 @@ class BooleanConfig extends React.Component {
               id={this.props.option.key}
               onChange={this._handleChange.bind(this)}
               onClick={this._stopBubble.bind(this)}
-              type="checkbox" />
-            <label
-              className={classes}
-              htmlFor={this.props.option.key}>
-              <div className="boolean-config--handle"></div>
+              type="checkbox"
+            />
+            <label className={classes} htmlFor={this.props.option.key}>
+              <div className="boolean-config--handle" />
             </label>
           </div>
         </div>
         <div
           className="boolean-config--description"
-          dangerouslySetInnerHTML={{__html: this.props.option.description}}>
-        </div>
+          dangerouslySetInnerHTML={{__html: this.props.option.description}}
+        />
```

Comment:
Ok, I really like that it's catching empty sections and converting them to self-closing bits.